src/utils.js

```diff
@@ -13,8 +13,17 @@ export const getComponentVariant = (theme, componentName, variant) => {
 
 export const getComponentStyle = componentName => themeGet(`components.${componentName}.style`);
 
-export const getComponentClassName = ({ className, theme: { classPrefix }, variant }, name) =>
-  `${className || ''} ${classPrefix}-${name} ${variant ? `${classPrefix}-${name}-${variant}` : ''}`.trim();
+const getComponentClassName = flow(
```

Comment:
Could the classnames package be helpful for managing some of this logic?

Reply:
It definitely could be (works really awesome for this particular use case) but I'm not sure pulling in another dependency just for this function would be worth it. Thoughts?

Reply:
Oh, I assumed we were already using it. Yes, we probably wouldn't want to pull it in to use only once. There's a [dedupe](https://github.com/JedWatson/classnames#alternate-dedupe-version) feature that seems like it could be helpful.